pkg/apis/network/v1/egressrouter_types.go

```diff
@@ -0,0 +1,273 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EgressRouter is a feature allowing the user to define an egress router
```

Comment:
lower-case. These are docs for JSON, not Golang types

Reply:
It gave me a "go-lint" suggestion to match it with the struct name when I make it small, that's why the R is in caps.